explainer/index.bs

```diff
@@ -23,19 +23,59 @@ Issue(tabatkins/bikeshed#2006): Set up cross-linking into the WebGPU and WGSL sp
 Issue(gpuweb/gpuweb#1321): Complete the planned sections.
 
 
-# Motivation # {#motivation}
+# Introduction # {#introduction}
 
-See [Introduction](https://gpuweb.github.io/gpuweb/#intro).
+WebGPU is a proposed Web API to enable webpages to use the system's [GPU (Graphics Processing Unit)](https://en.wikipedia.org/wiki/Graphics_processing_unit) to perform computations and draw complex images that can be presented inside the page.
+This goal is similar to the [WebGL](https://www.khronos.org/webgl/) family of API, but WebGPU enables access to more advanced features of GPUs.
+Whereas WebGL is mostly for drawing images but can be repurposed (with great effort) to do other kinds of computations, WebGPU has first-class support for performing general computations on the GPU.
 
-Issue: This section hasn't been written;
-[more actual motivation](https://w3ctag.github.io/explainers)
-is needed (here, in the spec Introduction section, or both).
+## Use cases ## {#use-cases}
 
+Example use cases for WebGPU that aren't addressed by WebGL 2 are:
+
+ - Drawing images with highly-detailed scenes with many different objects (such as CAD models). WebGPU's drawing commands are individually cheaper than WebGL's.
+ - Execute advanced algorithms for drawing realistic scenes. Many modern rendering techniques and optimizations cannot execute on WebGL 2 due to the lack of support for general computations.
+ - Execute machine learning models efficiently on the GPU. WebGL can be massaged to execute machine learning but it is difficult and sub-optimal.
+ - Provide a foundation to expose modern GPU functionality on the Web. WebGPU exposes functionality that became available in native around 2010 but is structured similarly to all current native GPU APIs. There are plans to later extend it to have more modern functionality. See also: [[#why-not-webgl3]].
```

Comment:
this reads wrong. Was it meant to talk about WebGL and not WebGPU?

Reply:
Can you detail what reads wrong? I'm not sure I understand.

Reply:
It says "WebGPU exposes functionality that became available in native around 2010".
In 2010 there were not any pipelines or passes, so I assume this sentence was meant to talk about WebGL instead.

Reply:
This was referring to compute shaders but extremely unclear. Removed that mention and reworded a bit.